src/CoreGraphics/CGColor.cs

```diff
@@ -330,6 +330,20 @@ static public CGColor CreateCmyk (nfloat cyan, nfloat magenta, nfloat yellow, nf
 			var h = CGColorCreateGenericCMYK (cyan, magenta, yellow, black, alpha);
 			return h == IntPtr.Zero ? null : new CGColor (h, owns: true);
 		}
+
+		[iOS (14,0)][TV (14,0)][Watch (7,0)][Mac (10,16)]
+		[DllImport(Constants.AccessibilityLibrary)]
+		static extern /* NSString */ IntPtr AXNameFromColor (/* CGColorRef */ IntPtr color);
```

Comment:
it's not named `Get` or `Copy` so the memory ownership is unclear
was that mentioned in the header files ? if so a comment would be nice :)

Reply:
Nothing